### PR TITLE
[#1296] Adding multiple sources now no longer changes the source of the ...

### PR DESCRIPTION
**Pull request by mjschranz**
...video to the newly added source

https://webmademovies.lighthouseapp.com/projects/65733/tickets/1296-fix-current-source-issue-when-multiple-sources-are-added

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @dseif received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/b049829b1981d45/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/b049829b1981d45/output.txt

Total script time: 1.01 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @dseif received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/10d9144f28b3846/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/10d9144f28b3846/output.txt

Total script time: 0.99 mins

Server is running on http://pr361.butyr.org/templates/test.html

File: src/core/views/media-view.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ define( [ "ui/page-element", "ui/logo-spinner", "util/lang", "ui/widget/textbox"
       _subtitle.classList.add( "form-ok" );
       _subtitle.classList.remove( "form-error" );
 
-      for ( var i = textboxes.length - 1; i >= 0; i-- ) {
+      for ( var i = 0; i < textboxes.length; i++ ) {
```
Comment (dseif) [inline src/core/views/media-view.js]: One small nit, can you store textboxes.length in a variable so we don't have to check what length is each time

**Pull request merged**